src/store.ts

```diff
@@ -214,10 +214,10 @@ const createAppOnboardingData = (data: FirestoreOnboardingUserData): AppOnboardi
   // TODO: take into account multiple colleges
   gradYear: data.gradYear ? data.gradYear : '',
   entranceYear: data.entranceYear ? data.entranceYear : '',
-  college: data.colleges[0].acronym,
+  college: data.colleges.length !== 0 ? data.colleges[0].acronym : undefined,
   major: data.majors.map(({ acronym }) => acronym),
   minor: data.minors.map(({ acronym }) => acronym),
-  grad: data.grad.map(({ acronym }) => acronym),
+  grad: data.grad.length !== 0 ? data.grad[0].acronym : undefined,
```

Comment:
data.grad does not seem to be defined when I test the staging site and don't have the `grad` field in Firestore (emulating our current users). Maybe checking if data.grad is undefined before checking its length will fix this?
![Screen Shot 2021-05-16 at 9 55 12 PM](https://user-images.githubusercontent.com/54298311/118422529-64a54d00-b691-11eb-8fa8-016519608563.png)